webpack/scenes/Subscriptions/Manifest/SimpleContentAccess.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { Switch } from '@patternfly/react-core';
+import { OverlayTrigger, Tooltip, Icon } from 'patternfly-react';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+function SimpleContentAccess(props) {
+  const {
+    canToggleSimpleContentAccess,
+    simpleContentAccessValue,
+    enableSimpleContentAccess,
+    disableSimpleContentAccess,
+    setActionInProgress,
+  } = props;
+
+  const [isChecked, setChecked] = useState(simpleContentAccessValue);
+
+  useEffect(() => {
+    setChecked(simpleContentAccessValue);
+  });
```

Comment:
Should add a dependency array so this effect won't run after every render, but will only run when that prop changes:

```suggestion
  useEffect(() => {
    setChecked(simpleContentAccessValue);
  }, [simpleContentAccessValue]);
```

Reply:
come to think of it, `setChecked` should probably also be in the deps array